### DISCUSSION TITLE
wrong out of renumber

### DISCUSSION THREAD

**Discussion by z331225718** [scikit-rf/scikit-rf#992] [Q&A]
I have a S64P file ,and I renumber it with some new order, but  the result is not what I needed.
I try it with matlab snp2smp fuc, it works well.

**Comment by jhillairet** [comment]
Can you attach an example of your issue in order to be able to reproduce it?

**Reply by z331225718** [reply]
I modify the renumber fuc：

s_temp = npy.emty_like(self.s)
for i in range(len(from_ports)):
    for j in range(len(from_ports)):
        s_temp[:,from_ports[i],from_ports[j]] = self.s[:,to-ports[i],to_ports[j]]
self.s = s_temp

it works well for my s64p file.

sorry for the file cannot be provided Because my data is stored on a confidential server
